apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/service/state/impl/BlockChainInfoServiceImpl.java

```diff
@@ -96,4 +96,9 @@ public Block getLastBlock() {
         return blockchain.getBlocksByAccountStream(accountId, from, to, timestamp);
     }
 
+    @Override
+    public int getLastBlockTimestamp() {
```

Comment:
I'm not completely sure if 'wrapper method' is really needed ???
Usually two part of data are taken as pair of calls :
`lookupBlockchain().getLastBlock().getTimestamp()`
`lookupBlockchain().getLastBlock().getHeight()`
and is used underneath , I'm not sure if we need special method for one parts only. But I do no insist.

Reply:
I see your point, but I don't consider making lookupBlockchain() public in BlockChainInfoServiceImpl. So I keep it as it is now

Reply:
I mean code like:
blockChainInfoService.getLastBlock().getTimestamp(), 
blockChainInfoService.getHeight()